pkg/kv/kvserver/metrics.go

```diff
@@ -1888,7 +1899,11 @@ func newStoreMetrics(histogramWindow time.Duration) *StoreMetrics {
 		RangeLossOfQuorumRecoveries:                  metric.NewCounter(metaRangeLossOfQuorumRecoveries),
 
 		// Raft processing metrics.
-		RaftTicks:                 metric.NewCounter(metaRaftTicks),
+		RaftTicks: metric.NewCounter(metaRaftTicks),
+		RaftQuotaPoolPercentUsed: metric.NewHistogram(
+			// NB: this results in 64 buckets (i.e. 64 timeseries in prometheus).
```

Comment:
Where does 64 come from? Is it possible to do e.g. 10?

Reply:
No, the histogram is pretty limited. It's a `hdrhistogram` and doesn't let you pick the buckets and also you can't arbitrarily dial down the resolution. Obviously we'd prefer to have ten linear buckets here instead of the hdr-style spacing.